content/docs/tasks/policy-enforcement/rate-limiting/index.md

```diff
@@ -252,12 +399,20 @@ namespace.
 
 ## Cleanup
 
-1. Remove the rate limit configuration:
+1. Remove the `memquota` rate limit configuration, if using `memquota`:
 
     {{< text bash >}}
     $ kubectl delete -f @samples/bookinfo/policy/mixer-rule-ratings-ratelimit.yaml@
     {{< /text >}}
 
+    Or
+
+    Remove the `redisquota` rate limit configuration, if using `redisquota`:
+
+    {{< text bash >}}
+    $ kubectl delete -f @redisquota.yaml@
```

Comment:
Is this is a GitHub file, you need to put the full path within the @@ (relative to the root of the repo)
If this is not a GitHub file, remove the @@

Reply:
Fixed